lua/telescope/_extensions/file_browser/picker.lua

```diff
@@ -63,10 +89,23 @@ fb_picker.file_browser = function(opts)
   opts.cwd = opts.cwd and vim.fn.expand(opts.cwd) or cwd
   opts.path = opts.path and vim.fn.expand(opts.path) or opts.cwd
   opts.files = vim.F.if_nil(opts.files, true)
+  opts.select_buffer = vim.F.if_nil(opts.select_buffer, false)
+
+  -- process select_buffer option
+  local finder
+  if opts.files and opts.depth == 1 and opts.select_buffer then
```

Comment:
Nitpick, let's change the order here from least to most likely

`opts.select_buffer and opts.files and opts.depth == 1`

The latter two might be argued, but I guess it's more often that users would want to start in `folder_browser` mode as they are unaware of `opts.depth`.